src/components/SearchPage/SearchPageComponent.js

```diff
@@ -1,51 +1,33 @@
 import React, { useEffect, useState } from 'react';
-
-import { Typography } from '@material-ui/core';
-import Divider from '@material-ui/core/Divider';
 import IconButton from '@material-ui/core/IconButton';
-import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemIcon from '@material-ui/core/ListItemIcon';
-import ListItemText from '@material-ui/core/ListItemText';
-import { makeStyles } from '@material-ui/core/styles';
 import ArrowBackIcon from '@material-ui/icons/ArrowBack';
-import InsertDriveFileRoundedIcon from '@material-ui/icons/InsertDriveFileRounded';
-import { colors } from 'colors';
 import { useDispatch, useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 import { useDebouncedCallback } from 'use-debounce';
-
+import { colors } from 'colors';
 import { searchResultClickedAction } from 'components/NotesMindMap/NotesMindMapActions';
-
 import { searchQueryChangeAction } from '../SearchPage/SearchPageAction';
+import { NotFound } from 'components/NotFound/NotFoundComponent';
+import { SearchList } from 'components/SearchList/SearchList';
 
 const BackButtonWrapper = styled.div`
-  color: red !important;
+  background: ${colors.darkGray} !important;
   padding: 1rem;
 `;
 
-const useStyles = makeStyles(() => ({
-  list: {
-    width: '100%',
-    height: '100%',
-    paddingLeft: 16,
-  },
-  listItem: {
-    padding: 0,
-    height: 64,
-  },
-  icon: {
-    paddingLeft: 12,
-    paddingRight: 12,
-  },
-  svgIcon: {
-    color: colors.iconColor,
-  },
-}));
+const Container = styled.div`
+  color: red !important;
+  width: 100%;
+  height: 100%;
+`;
+
+const InputWrapper = styled.div`
+  padding-left: 20px;
```

Comment:
The Search page topbar styles are being changed in https://github.com/DarthVanger/NeuralNotes/issues/194 by @kadokan81 .
I think it's better to avoid changing any styles for search input or back button in this PR in order to prevent conflicts with the [PR](https://github.com/DarthVanger/NeuralNotes/issues/194) created by @kadokan81 for the Search page TopBar.